proto/cosmos/staking/v1beta1/authz.proto

```diff
@@ -24,7 +24,8 @@ message StakeAuthorization {
   }
   // Validators defines list of validator addresses.
   message Validators {
-    repeated string address = 1;
+    // TODO how to deal with repeated here?
+    repeated string address = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
```

Comment:
Just wonder if i may apply `cosmos.AddressString` annotation to `repeared string address`.